tests/unit/intermediate/test_infer_schema.py

```diff
@@ -0,0 +1,57 @@
+from unittest.mock import create_autospec
+
+import pytest
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.remorph.intermediate.infer_schema import InferSchema
+
+
+@pytest.fixture(autouse=True)
```

Comment:
```suggestion
@pytest.fixture
```
If it causes issues - it's very difficult to debug